FeedStoreChallenge/CoreDataManager/ManagedCache+CoreDataClass.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ManagedCache+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Errol on 27/08/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+public class ManagedCache: NSManagedObject {
+	static func find(in context: NSManagedObjectContext) throws -> ManagedCache? {
+		let request = NSFetchRequest<ManagedCache>(entityName: ManagedCache.className())
```

Comment:
Suggestion:
Entity name and class name potentially can be different. Thus it is better to use `ManagedCache.entity().name!`
Also since this is a static method `ManagedCache` can be omitted.
```suggestion
		let request = NSFetchRequest<ManagedCache>(entityName: entity().name!)
```

Reply:
Got it. Thanks. 
Modified as per suggestion.